src/app/search/search.component.ts

```diff
@@ -410,6 +420,39 @@ export class SearchComponent implements OnInit, OnDestroy {
       .catch((error) => console.log(error));
   }
 
+  /**
+   * Updates the results table when there is no search term
+   *
+   * @param {string} value the elastic search query
+   * @memberof SearchComponent
+   */
+  updateResultsTableSeparately(toolsQuery: string, workflowsQuery: string) {
+    let toolHits;
+    let workflowHits;
+    // Combine array of tools and workflows to filter into results table
+    ELASTIC_SEARCH_CLIENT.search({
+      index: 'tools',
+      type: 'entry',
+      body: toolsQuery,
+    })
+      .then((tools) => {
+        toolHits = tools.hits.hits;
+        ELASTIC_SEARCH_CLIENT.search({
+          index: 'tools',
+          type: 'entry',
+          body: workflowsQuery,
+        })
+          .then((workflows) => {
```

Comment:
I thought I reviewed this already, adding on to Gary's comment that you can use forkJoin. Are there 2 PRs in one here?